steps/microsoft-teams-adaptive-card/1.0.4/step.yml

```
@@ -0,0 +1,162 @@
+title: Microsoft Teams Adaptive Card Integration
```

```suggestion
title: Send Microsoft Teams message
```
Suggesting a slightly readable title, change can be deferred to a later release.